app/views/schools/on_boarding/descriptions/_form.html.erb

```diff
@@ -1,9 +1,9 @@
-<%- self.page_title = 'Enter a short description of your school' -%>
+<%- self.page_title = 'Write a short description of your school' -%>
```

Comment:
I've done more research around this and realised we need an H1 called 'School description' and then to add 'Enter a description of your school' just above the text field. 

This will require help from the devs because there are other components which we need to remove an update on the page, so I think we should do this update separately and remove this from the pull request.